.github/workflows/reusable_workflow.yaml

```diff
@@ -34,33 +34,34 @@ on:
       QUAY_PASSWORD:
         required: false
 
+env:
+  QUAY_TAG_ROOT: quay.io/${{ secrets.QUAY_NAMESPACE }}/${{ inputs.image_name }}
+  QUAY_IMG_EXP: 0
```

Comment:
Note that I did test this and it works -- Setting the expiration to `0` results in no expiration. So we make this the default, and just override in the conditional var setting step for dev below.

Reply:
The `0` value would benefit from an explanatory comment.